oci/oci.go

```diff
@@ -275,7 +275,7 @@ func (r *Runtime) ExecSync(c *Container, command []string, timeout int64) (resp
 func (r *Runtime) StopContainer(c *Container) error {
 	c.opLock.Lock()
 	defer c.opLock.Unlock()
-	if err := utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.path, "kill", c.name); err != nil {
+	if err := utils.ExecCmdWithStdStreams(os.Stdin, os.Stdout, os.Stderr, r.path, "kill", c.name, "KILL"); err != nil {
```

Comment:
Makes sense, this change LGTM then. +1 on making this work as intended, will open an issue to discuss whether the current behavior is still what we want.

> Did you mean SIGTERM?

Yeah, I guess so - anything kinder than [terminating the process with extreme prejudice](https://en.wikipedia.org/wiki/Terminate_with_extreme_prejudice) ;)